src/airtable/rounds/funding_rounds.js

```diff
@@ -177,40 +180,42 @@ const calculateWinningProposalsForEarmark = (
       const multiplierOceanNew = (totalPool - fundedTillNow) / sumWeights
 
       proposals.map((x) => {
-        if (x.fields.funded !== x.fields.maxFund) {
+        if (!x.isFunded()) {
           x.fields.funded = Math.min(
-            x.fields.maxFund,
-            x.fields.weight * multiplierOceanNew + x.fields.funded
+            x.fields.maxFund - x.fields.totalFund,
+            x.fields.weight * multiplierOceanNew + x.fundAmount()
```

Comment:
Ok, so basically:
- fund whatever the proposal deserves based on weight
- Or fund based on whatever is left over

maxFund was wrong, because it would be 8k, instead of the remainder

Great catch!